frontend/tests/playwright/msw/browser.ts

```diff
@@ -0,0 +1,5 @@
+import { setupWorker } from 'msw';
+
+import { handlers } from './handlers';
+
+export const mswBrowser = setupWorker(...handlers);
```

Comment:
Since we are hoping to start using msw's browser interceptor in the future, I suppose it's okay to leave `browser.ts` and `mockServiceWorker.js` unused files 👍

Reply:
Yeah, I basically I left this to not have to re-add it later